browser/playlist/playlist_service_factory.cc

```diff
@@ -123,7 +123,7 @@ PlaylistServiceFactory::PlaylistServiceFactory()
           "PlaylistService",
           BrowserContextDependencyManager::GetInstance()) {
   PlaylistDownloadRequestManager::SetPlaylistJavaScriptWorldId(
-      ISOLATED_WORLD_ID_CHROME_INTERNAL);
```

Comment:
for consistency could you also change this in `browser/playlist/test/playlist_download_request_manager_browsertests.cc`?

Reply:
Oh, yeah. I missed that one. Thanks!